SPEC.md

```diff
@@ -143,17 +143,32 @@ When a build tool reads an artifact, it should for each input file:
 Concatenate all records together and apply lexical ordering by line to create the GitBOM document of the
 output artifact.
 
-### GitBOM Document persistence by a Build Tool
+## Annex A
 
-When persisting an output artifact to a file system, the build tool should create a subdirectory of the directory to which the output artifact is being written of the form:
-```.bom/objects/``` and write out the GitBOM document to ```.bom/objects/${gitoid:0:2}/${gitoid:2:}``` where ```${gitoid}```
-is the git object id of the GitBOM document.
+Annex A documents known methods of persisting GitBOM Documents to various stores.
+### GitBOM Document persistence by a Build Tool to its local filesystem
 
-### Build tool persistence of related metadata
+If a build tool persists GitBOM information to its local filesystem, the build tool should write out the GitBOM Document to ```${GITBOM_DIR}/objects/${Artifact Identifier Type uri prefix with ':' replaced by '/'}/${GitBOMID:0:2}/${GitBOMID:2:}``` where ```${artifact id}``` is the GitBOM id in lowercase hexidecimal.
 
-When persisting an output artifact to a file system, if the build tool has additional metadata to persist that makes reference to the Artifact Dependency Graph (ADG),
-it should persist that metadata to a subdirectory of the directory to which the output artifact is being written of the form: ```.bom/metadata/${tool}/```.  Filenaming and subdirectory structure below that point is at the discretion of the build tool.
-## Annex A
+Example:
+
+```
+.adg/objects/gitoid/blob/sha1/0e/8efd4cdf0d5bafcfcae658c2662a73b199b301
```

Comment:
what semantic value does `gitoid/blob/` add to the path?

Reply:
summary after discussion in the meeting...

Ed clarified that this was proposed as a means to help avoid downstream users, who choose to use non-upstream-identified hash functions, from future namespace collisions were the same name to be adopted upstream for a variation of the hash function. such downstream users would be assumed to use a path such as `.adg/objects/my_namespace/blob/my_sha/xx/xxxxx`

Reply:
I suggest either of the following updates to the PR:
1. remove `gitoid/` from the path, and state that private usage of hash functions should be discouraged, or namespaced in the hash function name
2. include `gitoid/` in the path, and state that private usage of hash functions is encouraged through declaring a private namespace in place of the "gitoid" string.

Reply:
My tendency is to be precise.  Since the "gitoid:blob:sha1" is a different hash than "sha1".  Obviously using ':' as a character in a path segment is problematic, so substitution of ':' is performed.  If we just use 'sha1' we would find ourselves in the strange position of 'sha1' not being actual sha1, and someone using 'sha1' using a string that is not 'sha1'.  I suspect this would be quite confusing.